app/assets/stylesheets/mobile/mobile.scss

```diff
@@ -176,7 +176,7 @@ footer {
     border-radius: 3px 3px 0 0;
 
     border: {
-      bottom: 1px solid #ccc;
+      bottom: 1px solid $border-medium-grey;
```

Comment:
When you are already changing and adding variables... can we try to get a more generic naming? This border may not be grey and I'd rather have a generic name for the variables. Also, we should find a consent between grey and gray, I guess...